gui/services/migrations/0020_add_enable_smb1.py

```diff
@@ -0,0 +1,40 @@
+from django.db import migrations, models
+
+
+def move_sysctl_min_protocol(apps, schemaeditor):
+    tunables = apps.get_model('system', 'tunable')
+    smb1_sysctl = tunables.filter(tun_type='SYSCTL',
+                                  tun_var='freenas.services.smb.config.server_min_protocol',
+                                  tun_value='NT1')
+
+    if smb1_sysctl.exists():
+        cifs.cifs_srv_enable_smb1 = True
```

Comment:
You also  need to get cifs

apps.get_model('services', 'cifs').objects.order_by('-id')[0]